src/bin/main.rs

```diff
@@ -61,6 +69,8 @@ fn app(props: &AppProps) -> Html {
     // was executed after the execute button is pressed.
     let executed_line = js_sys::Array::new();
     let not_highlighted = js_sys::Array::new();
+    let curr_line = Rc::new(RefCell::new(0.0));
```

Comment:
Great point! I haven't quite figured this one out. I started out with `use_state` but ran into issues passing the handler as props to multiple components (since I need to be able to set it in one child and read it in another). However, `use_mut_ref` I think is good here for now since we force re-renders anyways.